### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by callumjriley3**
Hey @bigdogmitchyp would you be able to review my class methods in band.py and also my variable naming across my programs? Much appreciated!

### CODE REVIEW

File: prac_09/band.py
Comment (bigdogmitchyp) [inline prac_09/band.py:1]: great work!

File: prac_09/silver_service_taxi.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        self.fanciness = fanciness
+        self.price_per_km = Taxi.price_per_km * fanciness
+
+    def __str__(self):
+        """Return a string like a Taxi but with flagfall stated."""
+        return f"{super().__str__()} plus flagfall of ${self.flagfall}"
+
+    def start_fare(self):
+        """Begin a new fare."""
+        super().start_fare()
```
Comment (bigdogmitchyp) [inline prac_09/silver_service_taxi.py]: Surplus to requirement

File: prac_09/taxi_simulator.py
Comment (bigdogmitchyp) [inline prac_09/taxi_simulator.py:1]: good work!

File: prac_09/silver_service_taxi.py (hunk 1 above)
Comment (callumjriley3) [reply prac_09/silver_service_taxi.py]: Good catch! Fixed it now. 👍

**Pull request merged**